src/style.css

```
@@ -1 +1,30 @@
-/* add styles here */
+* {
+  margin: 0;
+  padding: 0;
+}
+
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url('../src/images/star.svg');
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+}
+
+.stars__star:nth-child(5) {
+  margin-right: 0;
+}
```

```suggestion
}

.stars__star:not(:last-child) {
  margin-right: 4px;
}
```
it can be shorter. and in case if will be another amount of stars - no need will be change styles here. it will work for any amount of stars